HolyshiftService/Migrations/20231001174239_InitialCreate.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace HolyShift.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterDatabase()
+                .Annotation("MySql:CharSet", "utf8mb4");
+
+            /*migrationBuilder.CreateTable(
```

Comment:
Doesn't need to be commented you can just leave it. And in another migration it can be deleted 3adi.